spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -1271,6 +1271,53 @@
     end
   end
 
+  context 'split doc auth flow' do
+    before do
+      allow(IdentityConfig.store).to receive(:use_vot_in_sp_requests).and_return(true)
+      allow(IdentityConfig.store).to receive(:doc_auth_selfie_capture_enabled).and_return(true)
+      allow(IdentityConfig.store).to receive(:doc_auth_separate_pages_enabled).and_return(true)
+      visit_idp_from_oidc_sp_with_ial2(biometric_comparison_required: true)
+      sign_in_and_2fa_user(@user)
+      complete_doc_auth_steps_before_document_capture_step
+    end
+
+    context 'when doc auth seperated pages flow is enabled and selfie is required',
```

Comment:
You're already in the context of the split flow.
```suggestion
    context 'when selfie is required',
```

Reply:
Actually, given that this context is the only thing in the parent context, you could get rid of this context.
¯\\\_(ツ)\_/¯